### ISSUE TITLE
Add support for snap system-wide certificates

### ISSUE THREAD

**Issue by tresf** [qzind/tray#1239]
Per: https://snapcraft.io/docs/system-options#heading--store-certs

Closing this should also close #1238 

A possible workaround until this is implemented:

```bash
# Adds QZ Tray's root-ca.crt to the snap system store:
sudo snap set system store-certs.qz="$(cat /srv/qz/ssl/root-ca.crt)"
```

... and to remove:

```bash
# Removes QZ Tray's root-ca.crt from the snap system store:
sudo snap unset system store-certs.qz
```

**Comment by tresf** [comment]
Closed via 84138a0.  This patch will be part of QZ Tray 2.2.4.

**Comment by tresf** [comment]
According to @Vzor-'s testing, this should be supported in the current master branch.  He did find a bug which was preventing it from working.   If it's determined to not work for all snap applications, please request a reopen.
